src/assets/images/index.js

```diff
@@ -0,0 +1,6 @@
+import IconCalculator from './icon-calculator.svg'
+import IconKarma from './icon-karma.svg'
+import IconSupervisor from './icon-supervisor.svg'
+import IconTeamBuilder from './icon-team-builder.svg'
+
+export { IconCalculator, IconKarma, IconSupervisor, IconTeamBuilder }
```

Comment:
Kind of weird your commit message, I understand the purpose but the message could refer to a "index.js" file of a specific resource.